Documentation/contributing.rst

```diff
@@ -19,20 +19,115 @@ You need to have the following tools available in order to effectively
 contribute to Cilium:
 
 - git
+- go
+  [https://golang.org/dl/]
 - go-swagger
-  `go get -u github.com/go-swagger/go-swagger/cmd/swagger`
+  ``go get -u github.com/go-swagger/go-swagger/cmd/swagger``
 - go-bindata
-  `go get -u github.com/jteeuwen/go-bindata/...`
+  ``go get -u github.com/jteeuwen/go-bindata/...``
+- VirtualBox (if not using libvirt)
+  [https://www.virtualbox.org/wiki/Downloads]
+- Vagrant
+  [https://www.vagrantup.com/downloads.html]
 
```

Comment:
Please mark them as optional, you can run cilium and the test suite fine without VMs, no? Would also use the reStructuredText syntax for rendering links.